src/high-level/abstract-tensor.lisp

```diff
@@ -245,6 +279,51 @@ If TARGET is not specified then a new tensor is created with the same element ty
     ;; choice, like integers.
     (binary-operator #'expt source1 source2 target)))
 
+(define-backend-function .max (source1 source2 &optional target)
```

Comment:
Can you use DEFINE-EXTENSIBLE-FUNCTION without specialization here so that one has the option to micro-optimize on tensor type in the future?